### PR TITLE
Feature xl/home screen personalization2

### PR DESCRIPTION
**Pull request by theodur**
## Description of Change

Merges home screen personalization changes into`develop`.

## Testing

- [ ] Tested on iOS 
- [ ] Tested on Android 

## PR Checklist

  **Reviewer:** Confirm the items below as you review

- [ ] PR is connected to issue(s)
- [ ] Tests are included to cover this change (when possible)
- [ ] No magic strings (All string unions follow the [Union -> Constant](https://github.com/department-of-veterans-affairs/va-mobile-app/blob/develop/VAMobile/src/constants/common.ts) type pattern)
- [ ] No secrets or API keys are checked in
- [ ] All imports are absolute (no relative imports)
- [ ] New functions and Redux work have proper TSDoc annotations

## For QA

[Run a build for this branch](https://github.com/department-of-veterans-affairs/va-mobile-app/actions/workflows/on_demand_build.yml)

### CODE REVIEW

File: VAMobile/detoxconfig.js
Comment (theodur) [inline VAMobile/detoxconfig.js:1]: Just linting changes

File: VAMobile/documentation/design/Components/Alerts and Progress/Alert.md
Comment (theodur) [inline VAMobile/documentation/design/Components/Alerts and Progress/Alert.md:1]: Nothing has even changed in this file? lol

File: VAMobile/documentation/design/Components/Buttons and links/Button.md
Comment (theodur) [inline VAMobile/documentation/design/Components/Buttons and links/Button.md:1]: Same with this and the other Markdown files, nothing has even changed

**Review by dumathane — APPROVED** [review]

**Review by TKDickson — APPROVED** [review]
Home screen personalization feature is approved to merge to develop

**Pull request merged**